.vitepress/theme/ko/OurVision.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+    <div class="our-vision">
+        <div class="content">
+            <div class="buttons-container">
+                <a href="https://github.com/planetarium/libplanet" target="_blank" class="button">
+                    <img src="/github.svg" alt="GitHub logo" class="icon" />
+                    <div class="button-text">
+                        <strong>Libplanet</strong>
+                        <p>나인크로니클에서 사용되는 블록체인 라이브러리</p>
+                    </div>
+                </a>
+                <a href="https://github.com/planetarium/lib9c" target="_blank" class="button">
+                    <img src="/github.svg" alt="GitHub logo" class="icon" />
+
+                    <div class="button-text">
+                        <strong>Lib9c</strong>
+                        <p>나인크로니클의 프로토콜을 확인해보세요</p>
+                    </div>
+                </a>
+                <a href="https://github.com/planetarium/NCIPs" target="_blank" class="button">
+                    <img src="/github.svg" alt="GitHub logo" class="icon" />
+
+                    <div class="button-text">
+                        <strong>NCIP</strong>
+                        <p>나인크로니클에 의견을 내고 싶다면 NCIP를 확인해보세요</p>
+                    </div>
+                </a>
+            </div>
+
+            <div class="description">
+                <h1>Planetarium Labs</h1>
+                <p>
+                    <a href="https://docs.nine-chronicles.com/introduction/planetarium-labs/team" target="_blank"
+                        class="highlight">Planetarium Labs</a>은 커뮤니티 중심의 Web3 게임 회사입니다.<br />
+                    저희는 플레이어가 창작하고, 공유하며, 소유하고, 보상받을 수 있는 게임 경험을 통해 확장 가능하고 의미 있는 영향을 제공하는 것에 몰두하고 있습니다.<br />
+                    만약 <a href="https://github.com/planetarium/NineChronicles" target="_blank"
+                        class="highlight">나인크로니클</a> 같은 게임을 만들고 싶으시다면 Libplanet을 확인해보세요.
+                </p>
+            </div>
```

Comment:
I don't think it's not important to advertise Planetarium Labs organization. When I look at this part, I got the feeling that it is Planetarium Labs' game rather than the community's game.

Of course, it is good to show what Nine Chronicles is made of.

Reply:
Agree.